.NET/Microsoft.Recognizers.Text.DateTime/Extractors/BaseTimePeriodExtractor.cs

```diff
@@ -35,6 +35,9 @@ public List<ExtractResult> Extract(string text, DateObject reference)
                 timePeriodErs = TimeZoneUtility.MergeTimeZones(timePeriodErs, config.TimeZoneExtractor.Extract(text, reference), text);
             }
 
+            //TODO: Quick fix to solve german morgen (morning) / morgen (tomorrow) ambiguity, move to config
+            if (text.Equals("morgen")) timePeriodErs.Clear();

```

Comment:
I also notice this ambiguity thing in some of the test cases, thanks for fixing.
But this fix seems a little weird to me. How about the text is "morgen." (with dot ending) or "morgen" is part of the query?
Also, we suggest use bracket after if statement in C#.

Reply:
Fixing it with a hard match in the base class isn't a nice way to do it, but the only way we found without changing the configs for all languages. This adresses cases where the first word ist "morgen" like "morgen abend" or "morgen früh", so there will never be a "." or other non word char between those words, as this would change the meaning. I will add the brackets.